scipy/special/add_newdocs.py

```diff
@@ -2263,7 +2326,42 @@ def add_newdoc(place, name, doc):
     """
     expm1(x)
 
-    exp(x) - 1 for use when `x` is near zero.
+    Compute ``exp(x) - 1 ``, for use when `x` is near zero.
```

Comment:
While we're here, might be worth changing the language a bit; to my mind "for use when `x` is near zero" makes it sound like the function *isn't* for use when `x` is far away from zero. Similarly with `exprel`.